test/test_ops.py

```diff
@@ -1624,6 +1624,11 @@ def test_one_hot(self):
     data = [[[1, 2, 3], [0, 3, 5]], [[1, 2, 3], [0, 3, 5]]]
     helper_test_op([], lambda: torch.nn.functional.one_hot(torch.tensor(data), 8), lambda: Tensor(data).one_hot(8), forward_only=True)
 
+  @unittest.skipIf(getenv("METAL", 0)== 0, "only run on METAL backend")
```

Comment:
I think bfloat16 tests in test_dtype.py is a better place to add this. What's special about uniform? Would be nice to have the minimum test case that shows the issue. Finally need to set expect failure instead of just letting it fail.

Reply:
Hey, I can make it simpler with maybe just like `Tensor([1])`, nothing special about uniform.

And isn't letting it fail correct? I'm assuming that these errors aren't the expected output? Shouldn't tinygrad somehow gracefully handle `bfloat16` on Metal backend?

If I make it expect failure, then the CI essentially reflects that when there are bugs, the CI should pass?